R/dvhydrograph-data.R

```diff
@@ -163,7 +180,7 @@ extendStep <- function(toPlot){
   isStep <- 'type' %in% names(toPlot) && toPlot[['type']] == "s"
   
   if(isStep){
-    toPlot$x <- c(toPlot$x,  tail(toPlot$x, 1) + 90000) #this is 2400, if changing to POSTLT, need to use lubridate
+    toPlot$x <- c(toPlot$x,  tail(toPlot$x, 1) + 87000) #this is 2400, if changing to POSTLT, need to use lubridate
```

Comment:
Talked to you about this but consider changing to 60*60*24 (86,400) which is the number of seconds in a day or making a variable containing the number of seconds in a day.